docs/docs/logical_separation.md

```diff
@@ -1,52 +1,41 @@
 # Supporting Logical Data Separation (Multi-tenancy)
 
-A common architecture in web applications is multi-tenancy through logical separation. The FlaskMuckApiView includes 
-a method, `get_base_query_kwargs`, that can be overridden to generate a set of arguments that will be passed to all queries. 
-This allows you to filter resources to only those that should be available to a user and prevent resources from leaking 
-between tenants. 
+A common architecture in web applications is multi-tenancy through logical separation. `FlaskMuckApiView` includes a method, `get_base_query_kwargs`, which can be overridden to generate a set of arguments passed to all queries. This method enables filtering of resources to those available to a user, preventing resource leakage between tenants.
 
-The `get_base_query_kwargs` should return a dictionary of keyword arguments. Those kwargs will
-be applied to CRUD operations at the SqlAlchemy level. For example the GET /<resource\>/ endpoint will generate 
-the query `Model.query.filter_by(**get_base_query_kwargs()).all()` instead of `Model.query.all()`. The keyword arguments
-will also be applied when creating or updating a model.
+The `get_base_query_kwargs` should return a dictionary of keyword arguments. These kwargs are applied to CRUD operations at the SqlAlchemy level. For example, the GET /<resource>/ endpoint will generate the query `Model.query.filter_by(**get_base_query_kwargs()).all()` instead of `Model.query.all()`. The keyword arguments are also applied when creating or updating a model.
```

Comment:
```suggestion
The `get_base_query_kwargs` should return a dictionary of keyword arguments. These kwargs are applied to CRUD operations at the SqlAlchemy level. For example, the GET /<resource\>/ endpoint will generate the query `Model.query.filter_by(**get_base_query_kwargs()).all()` instead of `Model.query.all()`. The keyword arguments are also applied when creating or updating a model.
```